libp2p/transports/wstransport.nim

```diff
@@ -53,7 +53,12 @@ method readOnce*(
   s: WsStream,
   pbytes: pointer,
   nbytes: int): Future[int] {.async.} =
-  let res = await s.session.recv(pbytes, nbytes)
+  let res =
```

Comment:
We probably want to handle all the exceptions as in https://github.com/status-im/nim-libp2p/blob/unstable/libp2p/stream/chronosstream.nim#L67, also maybe a good idea to move to `/streams`?

Reply:
Done

I like to keep it self-contained, especially since it's that small (<50 lines for the stream)
But maybe in the future if someone wants to use WsStream without the transport?

Reply:
Thanks! Yeah, thats the logic that the streams are their own thing, we might for example use them in other places, like codex/dagger. We're already using `BufferStream` for example.